packages/components/src/BrandMoment/BrandMoment.module.css

```diff
@@ -38,67 +34,73 @@ $breakpoint-xl: 1366px; // TODO: create an xl breakpoint in design-tokens and re
   --brand-moment-background-color: var(--color-red-100);
 }
 
+/*
 // --------------------------------
 // Reusable container
 // --------------------------------
+*/
 .container {
-  max-width: $layout-content-max-width;
+  max-width: var(--layout-content-max-width);
   margin: 0 auto;
-  padding: 0 $spacing-md;
+  padding: 0, var(--spacing-24);
```

Comment:
When placing in multiple values into a a shorthand property in CSS like padding we don't need to add commas - it might actually break the CSS on the other end as it could interpret `,` as a design value.